src/azure-cli/azure/cli/command_modules/rdbms/_params.py

```diff
@@ -240,9 +240,9 @@ def _flexible_server_params(command_group):
                 c.argument('zone', options_list=['--zone, -z'],
                            help='Availability zone into which to provision the resource.')
             elif command_group == 'mysql':
-                c.argument('tier', default='Burstable', validator=tier_validator,
+                c.argument('tier', default='Burstable',
                            help='Compute tier of the server. Accepted values: Burstable, GeneralPurpose, Memory Optimized ')
-                c.argument('sku_name', default='Standard_B1ms', options_list=['--sku-name'], validator=mysql_sku_name_validator,
+                c.argument('sku_name', default='Standard_B1ms', options_list=['--sku-name'],
                            help='The name of the compute SKU. Follows the convention Standard_{VM name}. Examples: Standard_B1ms, Standard_D4s_v3 ')
```

Comment:
Is it possible to get allowed sku values through SDK?

Reply:
> HI @arde0708, as what I see in the e-mail, you need to fix the issue when creating a new flexible-server with HA in a VNET. May I know how you fix it?

@Juliehzl : All those changes are in src/azure-cli/azure/cli/command_modules/rdbms/flexible_server_virtual_network.py. We are basically ensuring that the VNET and Subnet Name do not contain the servername to make this fix.

Reply:
We have a permanent fix for this in S176 where we fetch the SKU's from the SDK. We are currently removing the validators in the PR as there is a bug in the validator which prevents provisioning servers with certain SKU's

Reply:
sounds good